create.js

```diff
@@ -31,6 +31,9 @@ function create (modules, options) {
     var data = {}
     for (var i = 0, max = names.length; max > i; i++) {
       var name = names[i]
+      if (input[name] === 'false') {
```

Comment:
I believe this has broken the case where we want explicit `"false"` values for attributes. e.g.,

```js
html`<img src="..."  draggable="false" />`
```

we end up throwing out the false attribute. This is what gets rendered:
```html
<img src="..." />
```